.github/workflows/lektor-check.yml

```diff
@@ -0,0 +1,39 @@
+name: lektor check
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.5, 3.6, 3.7, 3.8]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+    - name: Cache lfs
```

Comment:
This step name seems to be a duplicate.

Reply:
Ein update von pip tut nicht weh!

Reply:
Nicht die Zeile darüber, sondern diese. Der Name des Steps ist der Gleiche wie der darunter.

Reply:
dann markiere sie doch bitte korrekt.

Protipp: Es gehen auch multiline-markierungen!

Reply:
ja gut, das sollte - wie man sieht - lektor heißen.